src/fal/cli/flow_runner.py

```diff
@@ -89,15 +89,27 @@ def _run_sub_graph(
         _mark_dbt_nodes_status(fal_dbt, NodeStatus.Success, dbt_nodes)
```

Comment:
This needs to change to mark with real results. Also, we need to not raise errors until AFTER the post hooks.